README.rst

```diff
@@ -66,19 +76,20 @@ Usage
 ::
 
   $ python
+  >>> import os
   >>> from pysequila import SequilaSession
+  >>> curr_dir = os.getcwd()
   >>> ss = SequilaSession \
     .builder \
     .config("spark.jars.packages", "org.biodatageeks:sequila_2.12:1.1.0") \
```

Comment:
Outdated version

Reply:
There are some errors when I try to use a newer version, this one is the newest that works for me as of now.

Reply:
Then pls create an issue -  so that we keep track of it.